File: java/05 - Lab 3 - PetShop/src/test/java/org/cegeka/petshop/ShopTest.java
package org.cegeka.petshop;

import static org.assertj.core.api.Assertions.assertThat;

import java.util.Set;

import org.junit.Before;
import org.junit.Test;

public class ShopTest {

	private Shop shop;

	@Before
	public void setup() {
		shop = new Shop();
	}

	@Test
	public void addItem_whenItem_thenAddsItemToStock() {
		String item = "Doggy Gucci bag";

		shop.addItem(item);

		Set<String> items = shop.getItemsInStock();
		assertThat(items.contains(item)).isTrue();
	}
}

File: csharp/05 - Lab 3 - PetShop/PetShop.Tests/ShopTest.cs
using Xunit;
using FluentAssertions;

namespace PetShop.Tests
{
    public class ShopTest
    {
        private Shop _shop;

        public ShopTest() {
            _shop = new Shop();
        }

        [Fact]
        public void AddItem_whenItem_thenAddsItemToStock()
        {
            string item = "Doggy Gucci bag";
            _shop.AddItem(item);

            _shop.ItemsInShop.Should().Contain("Doggy Gucci bag");
        }
    }
}
